build.gradle

```diff
@@ -27,11 +27,33 @@ checkstyleMain {
 
 test.finalizedBy("checkstyleMain")
 
-task runHelloWorldJoseG (dependsOn: 'classes', type: JavaExec) {
-    main = 'jalau.at18.katas.helloworld.joseg.Main'
+task runHelloWorldMariaH (dependsOn: 'classes', type: JavaExec) {
+    main = 'jalau.at18.katas.helloworld.joseg.HelloWorld'
```

Comment:
This seems to be wrong, the task is `runHelloWorldMariaH` but it is pointing to `joseg.HelloWorld`. There should not be `runHelloWorldMariaH`, so I'm thinking that this change should be reverted